routes/TripsStack.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
```

Comment:
maybe drawer navigation?

Reply:
Sure, we can change it. I just wanted to finish but if we have time it can be changed

Reply:
This we can think about another time